module/VuFind/src/VuFind/Controller/LibraryCardsController.php

```diff
@@ -266,9 +300,14 @@ protected function processEditLibraryCard($user)
             $username = "$target.$username";
         }
 
+        $card = $user->getLibraryCard($id == 'NEW' ? null : $id);
```

Comment:
Should probably include a check that $id is not 'NEW' if $rename is true. This should be the case, but since the parameters come from the user, they should be validated.